_posts/2019-05-01-monthly-dev-post.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Monthly Dev Post of May 2019
+author: People
+---
+
+Welcome to our Monthly Dev Post of May 2019.
+Every month one of these post will be created, to tell you about the latest developments in the world of OpenTTD.
+
+<!-- more -->
+
+## Development
+
+### Highlights
+
+Development last month focused mostly on internal code changes and cleanup.
+Some of the user-visible changes are highlighted below.
+
+* Some isses were found in the 1.9 release and thus the [1.9.1 update](https://www.openttd.org/news/2019/04/08/openttd-1-9-1.html) was born.
+
+* A feature to to show the coverage area of built stations and the result of adding a station part was added.
+This meshes with the changes how the coverage area is defined that we taked about the the last dev post.
+
+* After a long, winding journey and several nearly-complete rewrites, NewRoadTypes (NRT) was finally merged.
```

Comment:
'NotRoadTypes', unless we're changing that (it might be confusing).

Reply:
The "not" prefix seems to be a weird internal meme only people who have been involved directly with the project for many years understand. I only have vague ideas where it might come from myself.